source/ID/concordium-did.rst

```diff
@@ -0,0 +1,546 @@
+- ``id`` - the DID of the account.
+- ``verificationMethod`` - the account's verification method.
+  It is a nested :ref:`threshold scheme <concordium-did-verification-method>` requiring at ``T`` out of ``M`` credentials to sign; each credential uses its own threshold scheme requiring ``R_i`` out of ``N_i`` keys to sign, where ``i = 1..M``.and ``j = 1..N_i``.
+  The credentials are identified by a `DID fragment`_ ``#credential-i``, and the keys in each credentials by ``#key-j-i`` where ``i = 1..M`` and ``j = 1..N_i``.
+- ``authentication`` - authentication method for the account.
+
+The document MAY include any other public data of a Concordium account.
+
+.. note::
+
+  A `DID fragment`_ allows for referencing a particular credential, or a key in the Account DID Document.
+  The fragment is used to locate the (unique) JSON object by matching the DID URL with the object's ``id`` property.
+
+.. seealso::
+
+  `Dereferencing a DID URL`_ in the W3C Credentials Community Group draft report.
+
+
+.. code-block:: json
+
+  {
+    "@context": [
+      "https://www.w3.org/ns/did/v1",
+      "Concordium DID URI"
+    ],
+    "id": "did:ccd:NET:acc:ADDR",
+    "verificationMethod": [
+      {
+        "id": "did:ccd:NET:acc:ADDR#acc-1",
+        "controller": "did:ccd:NET:acc:ADDR",
+        "type": "VerifiableCondition2021",
+        "blockchainAccountId": "ADDR",
+        "threshold": "T",
+        "conditionThreshold": [
+          {
+            "verificationMethod": [
+              {
+                "id": "did:ccd:NET:acc:ADDR#credential-1",
+                "controller": "did:ccd:NET:acc:ADDR",
+                "type": "VerifiableCondition2021",
+                "threshold": "R_1",
+                "conditionThreshold": [
+                  {
+                    "id": "did:ccd:NET:acc:ADDR#key-1-1",
+                    "type": "Ed25519VerificationKey2020",
+                    "controller": "did:ccd:NET:acc:ADDR#credential-1",
+                    "publicKeyMultibase": "XX"
+                  },
+                  "...",
+                  {
+                    "id": "did:ccd:NET:acc:ADDR#key-N_1-1",
+                    "type": "Ed25519VerificationKey2020",
+                    "controller": "did:ccd:NET:acc:ADDR#credential-1",
+                    "publicKeyMultibase": "YY"
+                  }
+                ]
+              }
+            ]
+          },
+          "...",
+          {
+            "verificationMethod": [
+              {
+                "id": "did:ccd:NET:acc:ADDR#credential-M",
+                "controller": "did:ccd:NET:acc:ADDR",
+                "type": "VerifiableCondition2021",
+                "threshold": "N",
+                "conditionThreshold": [
+                  {
+                    "id": "did:ccd:NET:acc:ADDR#key-1-M",
+                    "type": "Ed25519VerificationKey2020",
+                    "controller": "did:ccd:NET:acc:ADDR#credential-M",
```

Comment:
Why is this a controller? According to the spec this looks to be
```
An entity that has the capability to make changes to a [DID document](https://www.w3.org/TR/did-core/#dfn-did-documents). 
```

But why does credential-M have this capability? On its own it does not.

Reply:
`credential-M` is the controller for the public key that belongs to this credential. Maybe I don't fully understand the idea. I thought that each credential (as part of an account) controls its keys.

Reply:
Well it does in the sense that the keys cannot be changed without the credential signing. But that is on its own not enough. So in that sense, if the credential is part of a multi-credential account it has to have cooperation of others to change the keys.

Reply:
Ok, then is makes sense to make the account `did:ccd:NET:acc:ADDR` to be the controller.